1CAK/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+    clientId: "634332519398899724",
+    mediaKeys: false
+});
+let presenceData: presenceData = {
+    largeImageKey: "logo"
+};
+
+presence.on("UpdateData", async () => {
+    let startTimestamp: number = Date.now();
+    presenceData.startTimestamp = startTimestamp;
+    if (document.location.pathname === "/") {
+        presenceData.details = "Home Page";
+    } else if (document.location.pathname === "/trending") {
+        presenceData.details = "Trending Page";
+    } else if (document.location.pathname === "/recent") {
+        presenceData.details = "Recent Page";
+    } else if (document.location.pathname === "/legendary") {
+        presenceData.details = "Legendary Page";
+    } else if (document.location.pathname === "/friends") {
+        presenceData.details = "Friends Page";
+    } else if (document.location.pathname === "/rules") {
+        presenceData.details = "Reading the rules";
+    } else if (document.location.pathname === "/notifications") {
+        presenceData.details = "Notifications Page";
+    } else if (document.location.pathname === "/weeklytop" || document.location.pathname === "/alltimetop") {
+        presenceData.details = "Top Users";
+    } else if (document.location.pathname === "/preferences" || document.location.pathname === "/privacy_setting") {
```

Comment:
Maybe use switch case here?

Reply:
should i? i already sleep, may i will do it tomorrow

Reply:
k done.